src/index.css

```diff
@@ -31,18 +31,27 @@ body {
   width: 750;
 }
 
+.donutchart-container {
+  width: max-content;
+  gap: 2em!important;
+}
+.donutchart-graph {
```

Comment:
Why did these need to be updated?

Reply:
Those were there while I was testing preview. I've removed them and updated default style for donutchart-container itself.